system/Database/SQLite3/Table.php

```diff
@@ -227,6 +228,54 @@ public function dropForeignKey(string $foreignName)
         return $this;
     }
 
+    /**
+     * Adds primary key
+     */
+    public function addPrimaryKey(array $fields): Table
+    {
+        $primaryIndexes = array_filter($this->keys, static fn ($index) => strtolower($index['type']) === 'primary');
+
+        // if primary key already exists we can't add another one
+        if ($primaryIndexes !== []) {
+            return $this;
+        }
+
+        // add array to keys of fields
+        $pk = ['fields' => $fields['fields'],
+            'type'      => 'primary',
+        ];
```

Comment:
Not sure what CS Fixer would say about it, but it looks strange.
```suggestion
        $pk = [
            'fields' => $fields['fields'],
            'type'   => 'primary',
        ];
```